docs/book/v3/features/container/factories.md

```diff
@@ -505,6 +506,8 @@ It consumes the following `config` structure:
 ]
 ```
 
+- `extension` parameter available since `zendframework/zend-expressive-zendviewrenderer` 2.2.0
```

Comment:
Can you set the version at the beginning?

> Since 2.2.0: `extension` parameter...

This would be consistent with most version informations in the documentation. Thanks!

Reply:
Wouldn't it be confusing since this is zend-expressive documentation and release mentioned is from different package?

```suggestion
- Since ZendViewRenderer 2.2.0: `extension` parameter is used to configure default suffix used by NamespacedPathStackResolver 
```

Reply:
There is an issue open for splitting the documentation: #594